src/vs/platform/workspace/common/workspace.ts

```diff
@@ -89,7 +100,8 @@ export class WorkspaceContextService implements IWorkspaceContextService {
 	constructor(workspace: IWorkspace) {
 		this.workspace = workspace;
 		this.workspaceEmitter = new Emitter<IWorkspace>();
-		this.workspaceRegistry.set(this.workspace.resource.toString(), this.workspace);
+		const workspaceRegistryKey = workspace.resource.with({ fragment: '', query: '' }).toString();
```

Comment:
We don't want to be doing this in the long run, please make a TODO that when I change the URI scheme on Sourcegraph we no longer need to do `.with({ fragment: '', query: '' })`. Why do you need to here (just curious)?